### DISCUSSION TITLE
I wantt shaderrrrrss

### DISCUSSION THREAD

**Discussion by Theoyeah** [DEMOLITIONDON96/Demolition-Engine#66] [Source Show & Tell]
I TRIEDDD BUUTT THE GUY FORGORRR TO GIVE FUNKKINN.LUA HAAAAAAAAAAAAAAAA

**Comment by ThatOneSillyGuyDon** [comment]
I just added them, back

**Reply by Theoyeah** [reply]
thhaannnksss

**Reply by Theoyeah** [reply]
@DEMOLITIONDON96 WTF????

![image](https://user-images.githubusercontent.com/97792861/164501715-7a3467bb-5254-47fb-a613-c04b8ce4394c.png)


It don't work
